interface/src/app/gene/gene-network.tpl.html

```diff
@@ -3,8 +3,19 @@ <h1 class="page-header">
     Gene Network
   </h1>
   <div class="well">
-    <geneset-search-form ng-if="userObj"></geneset-search-form>
```

Comment:
I agree that having one directive instead of a bunch of code is more reusable. I must say, though, that the code that I added underneath is not for the geneset search form itself, but just to go to the geneset search form. Currently, this means opening up a modal where the geneset search form will be displayed. Do you think this approach makes sense?